src/FirmwareAnalysis/FirmwareAnalysis.Autorest/README.md

```diff
@@ -0,0 +1,88 @@
+
+## Info
+- Modifiable: yes
+- Generated: all
+- Committed: yes
+- Packaged: yes
+
+---
+## Detail
+This module was primarily generated via [AutoRest](https://github.com/Azure/autorest) using the [PowerShell](https://github.com/Azure/autorest.powershell) extension.
+
+## Module Requirements
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 2.7.5 or greater
+
+## Authentication
+AutoRest does not generate authentication code for the module. Authentication is handled via Az.Accounts by altering the HTTP payload before it is sent.
+
+## Development
+For information on how to develop for `Az.FirmwareAnalysis`, see [how-to.md](how-to.md).
+<!-- endregion -->
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+```yaml
+# pin the swagger version by using the commit id instead of branch name
+commit: ef348fed285ae01b78cf6afd394ad2c4c8b6da7e
+require:
+# readme.azure.noprofile.md is the common configuration file
+  - $(this-folder)/../../readme.azure.noprofile.md
+# - /swagger/fist/resource-manager/readme.md
+# If the swagger has not been put in the repo, you may uncomment the following line and refer to it locally
+# - (this-folder)/relative-path-to-your-local-readme.md
+
+input-file:
+  - $(repo)/specification/fist/resource-manager/Microsoft.IoTFirmwareDefense/stable/2024-01-10/firmwares.json
+  - $(repo)/specification/fist/resource-manager/Microsoft.IoTFirmwareDefense/stable/2024-01-10/workspaces.json  
+
+try-require: 
+  - /swagger/fist/resource-manager/readme.powershell.md
+
+# For new RP, the version is 0.1.0
+module-version: 0.1.0
+# Normally, title is the service name
+title: FirmwareAnalysis
+subject-prefix: $(service-name)
+
+# The next three configurations are exclusive to v3, and in v4, they are activated by default. If you are still using v3, please uncomment them.
+# identity-correction-for-post: true
+# resourcegroup-append: true
+# nested-object-to-string: true
+
+directive:
+  # Following are common directives which are normally required in all the RPs
+  # 1. Remove the unexpanded parameter set
+  # 2. For New-* cmdlets, ViaIdentity is not required
+  # Following two directives are v4 specific
+  - where:
+      variant: ^(Create|Update)(?!.*?Expanded|JsonFilePath|JsonString)
+    remove: true
+  - where:
+      variant: ^CreateViaIdentity.*$
+    remove: true
+  # Follow directive is v3 specific. If you are using v3, uncomment following directive and comments out two directives above
+  #- where:
+  #    variant: ^Create$|^CreateViaIdentity$|^CreateViaIdentityExpanded$|^Update$|^UpdateViaIdentity$
+  #  remove: true
+
+  # Remove the set-* cmdlet
+  - where:
+      verb: Set
+    remove: true
+
+  - where:
+      parameter-name: Id
+    set:
+      default:
+        name: Id
+        description: Gets the Id from the current context.
+        script: 'str(uuid.uuid4())'
```

Comment:
The script is not supported by powershell

Reply:
Sorry. I have an updated commit which i didn't publish yet because of some blocker. So let me post it. Thanks

Reply:
added a commit to fix this